meetings.md

```diff
@@ -28,19 +28,19 @@ there. We adjust regularity and format of meetings based on time of
 year, member workload (inside and outside co-op), and needs of
 organization-wide initiatives.
 
-:calendar: **Schedule:** We currently meet every 2 weeks, alternating between
+📆 **Schedule:** We currently meet every 2 weeks, alternating between
 format of a short **standup** and a **full** meeting.
 
 #### Full
-:clock10: **Duration:** 60 minutes
-:memo: **Template:** [`link.hypha.coop/template`](https://link.hypha.coop/template)
+🕙 **Duration:** 60 minutes  
```

Comment:
```suggestion
🕙 **Duration:** 60 minutes\
```

Don't need to talk about this now, but fwiw, these are equally valid, and one might be more self-apparent (and won't look like a mistake in whitespace that some people are likely to fix or accidentally remove, often through editor automation)

Demo: https://gist.github.com/patcon/38f6930b29bd86cf2e42626f20b89a30#newlines-via-escaped-newline-at-end-of-line

Reply:
Would prefer waiting on this until we make our own style doc somewhere else?